src/main/java/pl/javastart/couponcalc/PriceCalculator.java

```diff
@@ -5,9 +5,53 @@
 public class PriceCalculator {
 
     public double calculatePrice(List<Product> products, List<Coupon> coupons) {
-        // TODO
-
-        return 0;
+        double sum = 0;
+        if (products == null) {
+            return 0;
+        } else if (coupons == null) {
```

Comment:
Oprócz wartości null weź też pod uwagę przypadek z pustyim listami, czyli np. `products.isEmpty`, warto to też uwzględnić w testach